tests/components/konnected/test_config_flow.py

```diff
@@ -0,0 +1,677 @@
+"""Tests for Konnected Alarm Panel config flow."""
+from asynctest import patch
+import pytest
+
+from homeassistant.components.konnected import config_flow
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_SSDP
+
+from tests.common import MockConfigEntry
+
+
+# pylint: disable=redefined-outer-name
+@pytest.fixture
```

Comment:
If we set the fixture name here and rename the function to something else we don't need to disable redefined-outer-name.
```py
@pytest.fixture(name="mock_panel")
async def mock_panel_fixture():
    ...
```

Reply:
very nice! updating